File: formation-angular/src/app/model/personne.ts
export class Personne {
  private _id: number;
  private _prenom: string;
  private _nom: string;

  constructor(id?: number, prenom: string = '', nom: string = '') {
    this._id = id;
    this._prenom = prenom;
    this._nom = nom;
  }

  /**
   * Getter id
   * @return {number}
   */
  public get id(): number {
    return this._id;
  }

  /**
   * Getter prenom
   * @return {string}
   */
  public get prenom(): string {
    return this._prenom;
  }

  /**
   * Getter nom
   * @return {string}
   */
  public get nom(): string {
    return this._nom;
  }

  /**
   * Setter id
   * @param {number} value
   */
  public set id(value: number) {
    this._id = value;
  }

  /**
   * Setter prenom
   * @param {string} value
   */
  public set prenom(value: string) {
    this._prenom = value;
  }

  /**
   * Setter nom
   * @param {string} value
   */
  public set nom(value: string) {
    this._nom = value;
  }
}

File: lombok/src/main/java/formation/akka/lombok/demo/Personne.java
package formation.akka.lombok.demo;

import java.util.ArrayList;
import java.util.List;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.Singular;
import lombok.ToString;
import lombok.experimental.FieldDefaults;

@FieldDefaults(level = AccessLevel.PRIVATE)
@Builder
@NoArgsConstructor
@AllArgsConstructor(access = AccessLevel.PRIVATE)
@Getter
@ToString(of = { "id", "prenom", "nom" })
@EqualsAndHashCode(of = { "id" })
public class Personne {
	private Integer id;
	@Setter
	private String prenom;
	@Setter
	private String nom;

	@Getter
	@Singular
	private List<Adresse> adresses = new ArrayList<Adresse>();

}
